seqr/views/utils/terra_api_utils_tests.py

```diff
@@ -128,6 +129,19 @@ def test_list_workspaces(self, mock_logger, mock_redis):
             _ = list_anvil_workspaces(user)
         self.assertEqual(str(ec.exception),
             'Error: called Terra API: GET /api/workspaces?fields=public,workspace.name,workspace.namespace got status: 401 with a reason: Unauthorized')
+        self.assertEqual(ec.exception.status_code, 401)
+
+        responses.add(responses.GET, url, status=403)
+        with self.assertRaises(PermissionDenied) as ec:
+            _ = list_anvil_workspaces(user)
+        self.assertEqual(str(ec.exception),
+                         'test_user got access denied (403) from Terra API: GET /api/workspaces?fields=public,workspace.name,workspace.namespace with reason: Forbidden')
+
+        responses.add(responses.GET, url, status=503)
+        with self.assertRaises(PermissionDenied) as ec:
+            _ = list_anvil_workspaces(user)
+        self.assertEqual(str(ec.exception),
+                         'Terra API Unavailable (503): GET /api/workspaces?fields=public,workspace.name,workspace.namespace with reason: Service Unavailable')
```

Comment:
You may also want to test the status code of the exception.

Reply:
`PermissionDenied`error class does not have a status code associated with it